tests/test_api.py

```
@@ -1723,3 +1723,800 @@ def test_is_api_key_set_false():
+def test_rtsps_streams_empty():
+    """Test RTSPSStreams with no streams."""
+    from uiprotect.api import RTSPSStreams
+
+    streams = RTSPSStreams()
+
+    assert streams.get_active_stream_qualities() == []
+    assert streams.get_inactive_stream_qualities() == []
+    assert streams.get_available_stream_qualities() == []
+
+
+def test_rtsps_streams_edge_cases():
```

Ruff (0.12.2)

Redefinition of unused `test_rtsps_streams_edge_cases` from line 2066

(F811)